LoopKitUI/Views/Settings Editors/TherapySettingsView.swift

```diff
@@ -409,7 +409,7 @@ extension TherapySettingsView {
     
     private var supportSection: some View {
         Section {
-            NavigationLink(destination: Text("Therapy Settings Support Placeholder")) {
+            NavigationLink(destination: DemoPlaceHolderView(appName: appName)) {
```

Comment:
It is unknown where this view will be used. So it is easier to just require the appName to be passed in instead of expecting it to be in the environment.